### PR TITLE
Jeremy/eng 1068a host scaffolding

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
Rebranding

## How to Test
- [ ] Visit /work/host/
- [ ] Click Tabs and try various screen sizes

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/pages/Work.tsx [hunk 1]
```diff
@@ -16,10 +20,14 @@ import Signup from './signup';
...
+import '../styled/customStyles.scss';
 
-import Header from 'components/work/Header';
 
-import '../styled/customStyles.scss';
+const HostWithZendesk = (props: RouterProps) => (
+  <ZendeskWebWidgetWrapper>
+    <Host {...props} />
+  </ZendeskWebWidgetWrapper>
+);
```
Comment (woeltjen) [inline src/pages/Work.tsx]: Hm, thinking it could get unwieldy to include this level of detail here (e.g. when other routes need wrappers). I like that `ZendeskWebWidgetWrapper` is not included in `Host.tsx`, that's a good separation of concerns; maybe we can introduce an intermediary component to handle this wrapping external to `Work.tsx`?

I'm picturing:

* /host/HostPage.tsx (currently Host.tsx in this PR)
* /host/Host.tsx (ZendeskWebWidgetWrapper around HostPage)
* /host/index.ts (just exports Host.tsx)

File: src/pages/host/Host.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            onClick={handleNewListingClick}>
+            Add New Listing
+          </Button>
+        </Col>
+      </Row>
+      <hr />
+      <Nav className="mb-5" tabs>
+        {[
+          {
+          to: '/work/host/bookings',
```
Comment (woeltjen) [inline src/pages/host/Host.tsx]: Small whitespace irregularity here

Comment (kcvan) [reply src/pages/host/Host.tsx]: 👀

Comment (kcvan) [reply src/pages/host/Host.tsx]: also, maybe can probably move this out to a `const` later?  might keep the jsx a little cleaner

**Comment by kcvan** [comment]
besides small comments scaffolding looks great!

File: src/pages/host/Host.tsx (hunk 2 above)
Comment (woeltjen) [reply src/pages/host/Host.tsx]: As a bonus, if we moved this to a `const`, I think we could also generate the `AuthenticatedRoute`s in the `Switch` using that same array (minor improvement. but keeps the `to` paths from falling out of sync, e.g. when we drop the `/work`)

File: src/pages/Work.tsx (hunk 1 above)
Comment (jerminatorhits) [reply src/pages/Work.tsx]: +1 very clean architecture!

File: src/pages/host/Host.tsx (hunk 2 above)
Comment (jerminatorhits) [reply src/pages/host/Host.tsx]: ahh very nice

File: src/pages/host/HostPage.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { Redirect, Route, Switch, NavLink as RRNavLink } from 'react-router-dom';
+import { Listing, CREATE_LISTING, GET_HOST_LISTINGS } from 'networking/listings';
+import { Container, Fade, Button, NavItem, NavLink, Nav, Row, Col } from 'reactstrap';
+import AuthenticatedRoute from 'HOCs/AuthenticatedRoute';
+import NotFound from '../notFound';
+import { compose, graphql } from 'react-apollo';
+import { getFriendlyErrorMessage } from 'utils/validators';
+
+const HostRoutes = [
```
Comment (woeltjen) [inline src/pages/host/HostPage.tsx]: Small point, but the naming here is inconsistent; UpperCamelCase makes this look like either a class or a React component, given the conventions we use elsewhere. I think `HOST_ROUTES` would be most consistent with the way we've named other top-level constants.

Comment (jerminatorhits) [reply src/pages/host/HostPage.tsx]: kk thanks! I was confusing the use of UPPERCASE with enums, which are in a sense "constants" themselves.
